episodes/additional-resources.md

```diff
@@ -0,0 +1,59 @@
+---
+title: 'Additional resources'
+teaching: 10
+exercises: 0
+---
+
+:::::::::::::::::::::::::::::::::::::: questions 
+
+- Where can I find further information?
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+::::::::::::::::::::::::::::::::::::: objectives
+
+- Find additional resources for image analysis / microscopy online
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+## Additional resources
+
+If you want to learn more about image processing and microscopy, we've linked 
+some useful resources below: 
+
+### Bioimage analysis
+- [image.sc forum](https://forum.image.sc/)
+- [Pete Bankhead's bioimage book](https://bioimagebook.github.io/)
```

Comment:
Again maybe a brief sentence about what this covers, good resource delving deeper in the principles discussed during this course.